week-2/G-conditionals-2/exercise-2.js

```diff
@@ -8,7 +8,12 @@
 */
 
 function studentPassed(grade) {
-
+  if (grade < 50) {
+    return "fail";
+  }
+  if (grade >= 50) {
```

Comment:
Could we make this code a little shorter? What if we try using a `else` statement?

Reply:
hi Mussie  I find the G exercise a bit confusing in terms there are 4 exercises in G conditionals. you could have pushed the 2 ones separately. that make them clearer to follow. but overall well done ;)